### PR TITLE
Feature/387 cleansing job street

### LINKED ISSUE #387: Example job 'cleansing street'
Following the implementation of #159, we should have an example data cleansing job that does various correction tasks to our example data, filters the validated from the invalid parts and does some common reject writing to a file. In a diagram something a la this:

```
Src -> Corr1 -> F1 -> Corr2 -> F2 -> Write success
                 \             /
                   \        /
                 Write rejects
```

### PR DESCRIPTION
**Pull request by ClaudiaPHI**
Fixes #387 

I added 2 jobs.  

-  "Customer Age Filtering.analysis.xml"  for Community users 
- "OrderDB Name and Phone Cleansing.analysis.xml" for Professional users

### CODE REVIEW

File: desktop/ui/src/main/resources/datacleaner-home/jobs/OrderDB Name and Phone Cleansing.analysis.xml [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+	</source>
+	<transformation>
+		<transformer>
+			<descriptor ref="Name Correction" />
+			<metadata-properties>
+				<property name="CoordinatesY">87</property>
+				<property name="CoordinatesX">236</property>
+			</metadata-properties>
+			<properties>
+				<property name="Connection" value="named:" />
```
Comment (ClaudiaPHI) [inline desktop/ui/src/main/resources/datacleaner-home/jobs/OrderDB Name and Phone Cleansing.analysis.xml]: Unfortunately I do not know which property I should add for "Connection". I did not find any example.

File: desktop/ui/src/main/resources/datacleaner-home/jobs/OrderDB Name and Phone Cleansing.analysis.xml [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+	</source>
+	<transformation>
+		<transformer>
+			<descriptor ref="Name Correction" />
+			<metadata-properties>
+				<property name="CoordinatesY">87</property>
+				<property name="CoordinatesX">236</property>
+			</metadata-properties>
+			<properties>
+				<property name="Connection" value="named:demo" />
```
Comment (ClaudiaPHI) [inline desktop/ui/src/main/resources/datacleaner-home/jobs/OrderDB Name and Phone Cleansing.analysis.xml]: I really hope this is ok. My guess it's not.

Comment (kaspersorensen) [inline desktop/ui/src/main/resources/datacleaner-home/jobs/OrderDB Name and Phone Cleansing.analysis.xml]: Check the common-datacleaner project. I think it's just "demo" but I cannot recall. It must be in the Converter class that handles ConnectionInformation conversion

**Comment by ClaudiaPHI** [comment]
Modified and tested. I think it can be merged

**Comment by LosD** [comment]
Minor thing: The other age-related example job uses entirely different captilization: "Customer age analysis.analysis.xml" and "Customer Age Filtering.analysis.xml". That looks a bit weird. You may also want to avoid the names being _that_ similar.

(In general, we may want to clean all of these up at some point. The jobs folder is pure chaos, with spaces, underscores, all lower case, mixed case, etc)

Another minor thing is that both jobs uses a "dump/invalid record" file that is in the middle of the graph. I'm not sure if it is something you should necessarily do anything about (and it may very well make the general layout worse), but in general I think it is a little easier to understand when output files are to the right.

**Comment by ClaudiaPHI** [comment]
I will do the file naming clean up.  

The dump file is at the bottom so that we emphasize the filter capabilities and so that we show that we  can put records into a csv file coming from more components.

**Comment by LosD** [comment]
Fine with me! :)

**Comment by ClaudiaPHI** [comment]
Can we merge this ?

**Comment by LosD** [comment]
Yep, when Travis is happy :)

**Pull request merged**